var/spack/repos/builtin/packages/protobuf/package.py

```diff
@@ -36,26 +35,34 @@ class Protobuf(CMakePackage):
     version('3.2.0',   sha256='a839d3f1519ff9d68ab908de5a0f269650ef1fc501c10f6eefd4cae51d29b86f')
     version('3.1.0',   sha256='fb2a314f4be897491bb2446697be693d489af645cb0e165a85e7e64e07eb134d')
     version('3.0.2',   sha256='a0a265bcc9d4e98c87416e59c33afc37cede9fb277292523739417e449b18c1e')
+    version('2.5.0',   sha256='c2665a7aa2ac1a206e61b28e014486e3de59009ea2be2bde9182e0847f38b62f')
 
     variant('shared', default=True,
             description='Enables the build of shared libraries')
     variant('build_type', default='Release',
             description='The build type to build',
             values=('Debug', 'Release'))
 
+    depends_on('cmake', when='@3.0.2:', type='build')
     depends_on('zlib')
+    depends_on('autoconf', type='build', when='@2.5.0')
+    depends_on('automake', type='build', when='@2.5.0')
+    depends_on('libtool',  type='build', when='@2.5.0')
+    depends_on('m4',       type='build', when='@2.5.0')
 
     conflicts('%gcc@:4.6', when='@3.6.0:')  # Requires c++11
     conflicts('%gcc@:4.6', when='@3.2.0:3.3.0')  # Breaks
 
     # first fixed in 3.4.0: https://github.com/google/protobuf/pull/3406
-    patch('pkgconfig.patch', when='@:3.3.2')
+    patch('pkgconfig.patch', when='@3.0.2:3.3.2')
 
     patch('intel-v1.patch', when='@3.2:@3.6 %intel')
 
     # See https://github.com/protocolbuffers/protobuf/pull/7197
     patch('intel-v2.patch', when='@3.7:@3.11.4 %intel')
 
+    patch('protoc.patch', sha256='7b44fcdb794f421174d619f83584e00a36012a16da09079e2fad9c12f7337451', when='@2.5.0 target=aarch64:')
```

Comment:
Tests are currently failing because this patch cannot be found, I think you'll want to use the name "protoc2.5.0_aarch64.patch".

Reply:
OK, I remember that I have changed the name, maybe it been overwrited again, sorry.

Reply:
@scheibelp  @adamjstewart 
Any more suggestion for this patch?